workspaces/ui-v2/src/optic-components/hooks/edit/Contributions.tsx

```diff
@@ -106,15 +111,30 @@ export const ContributionEditingStore: FC<ContributionEditingStoreProps> = (
         try {
           await spectacleMutator({
             query: `
-            mutation X($commands: [JSON], $commitMessage: String) {
-              applyCommands(commands: $commands, commitMessage: $commitMessage) {
+            mutation X(
```

Comment:
One variant of code that was missing some other updates here https://github.com/opticdev/optic/blob/develop/workspaces/ui-v2/src/optic-components/diffs/render/AskForCommitMessage.tsx#L65